crates/iota-indexer/src/test_utils.rs

```diff
@@ -21,6 +21,8 @@ use crate::{
     store::{PgIndexerAnalyticalStore, PgIndexerStore},
 };
 
+pub type DBInitHook = Box<dyn FnOnce(&PgIndexerStore) + Send>;
```

Comment:
Please add documentation with an example on the usage of this type.

Reply:
Done 563c23734fade7c5fb48a0db38f1337decfc3c39